src/compiler/checker.ts

```diff
@@ -8321,8 +8370,8 @@ namespace ts {
             function checkForDisallowedESSymbolOperand(operator: SyntaxKind): boolean {
                 let offendingSymbolOperand =
                     someConstituentTypeHasKind(leftType, TypeFlags.ESSymbol) ? node.left :
-                    someConstituentTypeHasKind(rightType, TypeFlags.ESSymbol) ? node.right :
-                    undefined;
+                        someConstituentTypeHasKind(rightType, TypeFlags.ESSymbol) ? node.right :

```

Comment:
@aozgaa, this repository has `.editorconfig` file at root. If you install editorconfig extension for your editor (available for all editors [including VS](https://visualstudiogallery.msdn.microsoft.com/c8bccfe2-650c-4b42-bc5c-845e21f96328)), it will format the documents accordingly (`Ctrl,K,D`). Editorconfig extension automatically apply your projects settings to editor scoped to that project only.